src/playermat/GenericCounter.ttslua

```diff
@@ -0,0 +1,39 @@
+MIN_VALUE = -99
+MAX_VALUE = 999
+val = 0
+
+function onSave() return JSON.encode(val) end
+
+function onLoad(saved_data)
+    if saved_data ~= nil then
+        val = JSON.decode(saved_data)
+    end
+
+    self.createButton({
+        label = tostring(val),
+        click_function = "addOrSubtract",
+        function_owner = self,
+        position = { 0, 0.06, 0 },
```

Comment:
The damage trackers all had slightly different positions to keep them in the right place given the different token shapes.  Making this common loses that.  Extract to a parent file for each tracker type?
![image](https://user-images.githubusercontent.com/5797523/202824531-f9614596-ce36-4c88-a2f7-cb4b8f52e097.png)

Reply:
Previous centering, for reference
![image](https://user-images.githubusercontent.com/5797523/202824918-0b171014-1884-4474-889f-648458625762.png)

Reply:
using name to distinguish between positions now